src/main/java/spoon/reflect/meta/impl/SingleHandler.java

```diff
@@ -27,7 +27,7 @@
 /**
  * implementation of {@link RoleHandler}, which handles attributes of type &lt;V&gt;
```

Comment:
You missed a V here.

Reply:
?

Reply:
It says that it handles attributes of type V but there is no type V defined in the class signature now.

Reply:
Class signature is: abstract class SingleHandler<T, U> extends AbstractRoleHandler<T, U, U>
so why 'V' should be documented?

Reply:
It should not but currently the comment is the following:
>[...] which handles attributes of type &lt;V&gt;

So V or `&lt;V&gt;` is currently documented and should not be.

Reply:
Ok, not it is not documented.